Yaml linting

aslape opened the pull request:
Saw some discussion in slack about yaml linting, so I figured I'd throw up a command for SDCLI. This works about how you'd expect--go to the project containing the yaml files you'd like to lint, and type: 

> sdcli yaml lint

I've run it against a few of our yamls already and we seem to already be pretty compliant, save for a common warning about not using a document start marker ("----") on line 1.

zlozano commented:
Question: will we need to add a .yamllint to each one of our projects? How would we go about turning off the document start marker?

aslape commented:
@zlozano It will run without the .yamllint file, and we can disable the document start marker warning in either a config file or with an argument in the command itself. I can go ahead and disable it in the command to save us the trouble of making 30 .yamllint files, if that sounds good.

aslape commented:
So I added a few new changes. 1) I updated the yaml linting command to disable looking for the document start string, and to enforce quoted string values in yaml files, and 2) I updated the yaml files in SDCLI to meet our new linter's requirements.
